### ISSUE TITLE
Randomizer won't run

### ISSUE THREAD

**Issue by Alice-Cheshire** [thezerothcat/LaMulanaRandomizer#96]
So this actually appears to be an issue with my Java install but I can't for the life of me figure it out. Hopefully you might have some insights here. When I try to run the randomizer I get the following errors:
[First error](https://i.imgur.com/2Q7XX2Z.png)
[When I click "okay" there](https://i.imgur.com/uCO2XT2.png)

I've tried completely uninstalling, redownloading, and reinstalling Java multiple times (multiple versions as well) but I still get this error no matter what. From what little I can find online it seems like some issue with Windows 10 but I can't find any solutions for it.

**Comment by dansidney** [comment]
You have to manually download the latest version of Java version 8 from their website.

**Comment by Alice-Cheshire** [comment]
As I said, I tried reinstalling it from multiple versions. Not only the default web installer but the latest manual installer and two previous versions of the manual installer as well. All resulted in the same issue.

**Comment by thezerothcat** [comment]
Java 8? No, you'd need Java 11 to run the randomizer on its current version.

How are you running the randomizer? If you are able to run from command line there would be more error information which could help for troubleshooting.

**Comment by Alice-Cheshire** [comment]
Okay, so it seems I did have the wrong version of Java. Java.com is the first result when you do a search for Java. The version that site has only up to Java 8 though. Oracle's site on the other hand goes up to Java 12. I had thought that Java.com was the official site for the language but it's either not updated or not actually the official site. Once I installed Java 12 from Oracle's site it started working perfectly fine.
